library/ix-dev/community/jellyfin/questions.yaml

```diff
@@ -0,0 +1,337 @@
+groups:
+  - name: Jellyfin Configuration
+    description: Configure Jellyfin
+  - name: User and Group Configuration
+    description: Configure User and Group for Jellyfin
+  - name: Network Configuration
+    description: Configure Network for Jellyfin
+  - name: Storage Configuration
+    description: Configure Storage for Jellyfin
+  - name: Resources Configuration
+    description: Configure Resources for Jellyfin
+
+portals:
+  web_portal:
+    protocols:
+      - "$kubernetes-resource_configmap_portal_protocol"
+    host:
+      - "$kubernetes-resource_configmap_portal_host"
+    ports:
+      - "$kubernetes-resource_configmap_portal_port"
+    path: "$kubernetes-resource_configmap_portal_path"
+
+questions:
+  - variable: jellyfinConfig
+    label: ""
+    group: Jellyfin Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: publishServerUrl
+          label: Published Server URL
+          description: Optional - The URL that Jellyfin will be advertised.
+          schema:
+            type: string
+            default: ""
+        - variable: additionalEnvs
+          label: Additional Environment Variables
+          description: Configure additional environment variables for Jellyfin.
+          schema:
+            type: list
+            default: []
+            items:
+              - variable: env
+                label: Environment Variable
+                schema:
+                  type: dict
+                  attrs:
+                    - variable: name
+                      label: Name
+                      schema:
+                        type: string
+                        required: true
+                    - variable: value
+                      label: Value
+                      schema:
+                        type: string
+                        required: true
+
+  - variable: jellyfinRunAs
+    label: ""
+    group: User and Group Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: user
+          label: User ID
+          description: The user id that Jellyfin will run as.
+          schema:
+            type: int
+            min: 1
+            default: 568
+            required: true
+        - variable: group
+          label: Group ID
+          description: The group id that Jellyfin will run as.
+          schema:
+            type: int
+            min: 1
```

Comment:
On second thought - do we really want some apps to be allowed to run as root when there is no usecase?
We can update min accordingly in that case?

Reply:
What min should we have here, `568`?

Reply:
Maybe yes?
We can revisit later if someone complains..

Reply:
Currently it can't run as root `0` (min is `1`).
I'll make a PR to mass update to `568` all new apps maybe?